test/policies/organizations/like_policy_test.rb

```diff
@@ -0,0 +1,215 @@
+require "test_helper"
+
+class Organizations::LikePolicyTest < ActiveSupport::TestCase
+  include PetRescue::PolicyAssertions
+
+  context "#index?" do
+    setup do
+      @organization = ActsAsTenant.current_tenant
+      @policy = -> { Organizations::LikePolicy.new(Like, user: @user, organization: @organization) }
+      @action = -> { @policy.call.apply(:index?) }
+    end
+
+    context "when user is nil" do
+      setup do
+        @user = nil
+      end
+
+      should "return false" do
+        assert_equal @action.call, false
+      end
+    end
+
+    context "when user is adopter" do
+      setup do
+        @user = create(:adopter)
+      end
+
+      should "return true" do
+        assert_equal @action.call, true
+      end
+    end
+
+    context "when user is fosterer" do
+      setup do
+        @user = create(:fosterer)
+      end
+
+      should "return true" do
+        assert_equal @action.call, true
+      end
+    end
+
+    context "when user is staff" do
+      setup do
+        @user = create(:staff)
+      end
+
+      should "return false" do
+        assert_equal @action.call, false
+      end
+    end
+  end
+
+  context "#create?" do
+    setup do
+      @organization = ActsAsTenant.current_tenant
+      @pet = create(:pet)
+      @policy = -> {
+        Organizations::LikePolicy.new(Like,
+          user: @user,
+          organization: @organization,
+          pet: @pet)
+      }
+      @action = -> { @policy.call.apply(:create?) }
+    end
+
+    context "when user is nil" do
+      setup do
+        @user = nil
+      end
+
+      should "return false" do
+        assert_equal @action.call, false
+      end
+    end
+
+    context "when user is adopter" do
+      setup do
+        @user = create(:adopter)
+      end
+
+      should "return true" do
+        assert_equal @action.call, true
+      end
+    end
+
+    context "when user is staff" do
+      setup do
+        @user = create(:staff)
+      end
+
+      should "return false" do
+        assert_equal @action.call, false
+      end
+    end
+  end
+
+  context "#create?" do
```

Comment:
This is a duplicate name of the above context. Should the tests in this block be moved into the one above?